src/components/listings/MasonryListItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { ImageSourcePropType } from 'react-native'
+import { Image } from 'expo-image'
+
+interface MasonryListItemProps {
+  image: ImageSourcePropType
+  containerWidth: number
+}
+
+function MasonryListItem(props: MasonryListItemProps) {
+  const { image, containerWidth } = props
+
+  const blurhash =
+    '|rF?hV%2WCj[ayj[a|j[az_NaeWBj@ayfRayfQfQM{M|azj[azf6fQfQfQIpWXofj[ayj[j[fQayWCoeoeaya}j[ayfQa{oLj?j[WVj[ayayj[fQoff7azayj[ayj[j[ayofayayayj[fQj[ayayj[ayfjj[j[ayjuayj['
+
+  const [imgHeight, setImageHeight] = useState<number>(190)
```

Comment:
const this as well please
`const IMG_HEIGHT_DEFAULT = 190`

Reply:
👍